cider/tests/benchmark/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
+find_package(Arrow REQUIRED)
```

Comment:
this will use velox provided arrow library?

Reply:
use installed package. it seems here where docker container installed arrow/parquet library.
https://github.com/intel/BDTK/blob/534e9bad307d46f878bc3d729111aa46f074f836/ci/docker/scripts-cider/common-functions.sh#L92-L102